draft-ietf-httpbis-digest-headers.md

```diff
@@ -77,442 +76,372 @@ normative:
-      ASCII decimal-digit string representing the 16-bit
-      checksum, which is the first word of the output of
-      the UNIX "sum" command.
-    * Reference: [UNIX], this document.
-    * Status: insecure
-
-  unixcksum
-  : * Digest Algorithm: unixcksum
-    * Description: The algorithm used by the UNIX "cksum" command [UNIX].
-      The output of this algorithm is an
-      ASCII digit string representing the 32-bit CRC,
-      which is the first word of the output of the UNIX
-      "cksum" command.
-    * Reference: [UNIX], this document.
-    * Status: insecure
-
-  adler32
-  : * Digest Algorithm: adler32
-    * Description: The ADLER32 algorithm [RFC1950].
-      The 32-bit output is encoded in hexadecimal (using
-      between 1 and 8 ASCII characters from 0-9, A-F, and a-f; leading 0's are
-      allowed). For example, adler32=03da0195 and adler32=3DA0195 are both valid
-      checksums for the 4-byte message "Wiki". This algorithm is obsoleted and
-      SHOULD NOT be used.
-    * Reference: [RFC1950], this document.
-    * Status: insecure
-
-  crc32c
-  : * Digest Algorithm: crc32c
-    * Description: The CRC32c algorithm {{!RFC4960}}.
-      The 32-bit output is encoded in hexadecimal (using between 1 and 8
-      ASCII characters from 0-9, A-F, and a-f; leading 0's are allowed). For
-      example, crc32c=0a72a4df and crc32c=A72A4DF are both valid checksums for the
-      3-byte message "dog".
-    * Reference: {{!RFC4960}} appendix B, this document.
-    * Status: insecure
-
-# Using Digest in State-Changing Requests {#state-changing-requests}
+The entries in {{iana-hash-algorithm-table}} are registered by this document.
+
+| -------------- | -------- | ----------------------------------- | -------------- |
+| Algorithm Key  | Status   | Description                         | Reference(s)   |
+| -------------- | -------- | ----------------------------------- | --------- |
+| sha-512        | standard | The SHA-512 algorithm.              | [RFC6234], [RFC4648], this document. |
+| sha-256        | standard | The SHA-256 algorithm.              | [RFC6234], [RFC4648], this document. |
+| md5            | insecure | The MD5 algorithm. It is vulnerable to collision attacks; see {{NO-MD5}} and [CMU-836068] | [RFC1321], [RFC4648], this document. |
+| sha            | insecure | The SHA-1 algorithm. It is vulnerable to collision attacks; see {{NO-SHA}} and [IACR-2020-014] | [RFC3174], [RFC4648], [RFC6234] this document. |
+| unixsum        | insecure | The algorithm used by the UNIX "sum" command. | [RFC4648], [RFC6234], [UNIX], this document. |
+| unixcksum      | insecure | The algorithm used by the UNIX "cksum" command. | [RFC4648], [RFC6234], [UNIX], this document. |
+| adler          | insecure | The ADLER32 algorithm.                          | [RFC1950], this document. |
+| crc32          | insecure | The CRC32c algorithm.                           | {{!RFC4960}} appendix B, this document. |
```

Comment:
```suggestion
| crc32c         | insecure | The CRC32c algorithm.                           | {{!RFC4960}} appendix B, this document. |
```

Reply:
I agree with the change of algorithm name, but I'll hold off merging to avoid the conversation being resolved and hard to find.